src/components/FlexibleSection/SectionButton.tsx

```diff
@@ -41,4 +46,7 @@ const useStyles = makeStyle((theme) => ({
     width: 24,
     marginRight: theme.spacing.s,
   },
+  disabled: {
+    opacity: 0.3,
```

Comment:
Instead of setting the opacity to `0.3`, can't we simply make the component return `null` instead? This way nothing will be displayed at all and we would avoid any issue. Also I think it might be better to change the attribute name from `isDisabled` to `isHidden` in this case as a disabled button usually has opacity 100% and simply can't be pressed

Reply:
The `FlexibleSection.Section` has been structured to have no hidden elements in it so that it handles `BottomBorder` correctly. I can comment out the element beforehand if you prefer to hide it.

Reply:
@ale-mazz Yes please just comment it out then